app/routes/redirect-list.js

```diff
@@ -2,13 +2,16 @@
 
 // maybe in the future this will come out of the environment, or a database, or a file or all
 
+import getSitemap from './sitemap'
 const list = [{ path: '/undebates', dst: 'https://cc.enciv.org/undebates' }]
 
 export default function redirectList() {
   for (const red of list) {
     this.app.get(red.path, (req, res, next) => {
       try {
-        res.redirect(red.dst)
+        getSitemap().then(() => {
```

Comment:
So if we are redirecting something to another site, is this site responsible for that in the sitemap? 
I think we should leave that to another time, and just get the basic site map working.

Also, I don't understand why we would get a sitemap in this case.

Reply:
Actually I dont get this part either. This approach is from our previous conversation. For me I prefer to just call getSitemap() when the server start and when bots visit "www.enciv.org/sitemap.xml"